addons/mail/static/src/models/call_main_view.js

```diff
@@ -117,12 +117,35 @@ registerModel({
         // Private
         //----------------------------------------------------------------------
 
+        /**
+         * @private
+         */
+        _computeHasSidebarButton() {
+            return Boolean(this.callView.activeRtcSession && this.showOverlay && !this.callView.threadView.compact);
+        },
         /**
          * @private
          */
         _computeIsControllerFloating() {
             return Boolean(this.callView.isFullScreen || this.callView.activeRtcSession && !this.callView.threadView.compact);
         },
+        /**
+         * @private
+         * @returns {FieldCommand}
+         */
+        _computeMainTiles() {
+            if (this.callView.activeRtcSession) {
+                return insertAndReplace([{ channelMember: replace(this.callView.activeRtcSession.channelMember) }]);
+            }
+            const tiles = [];
+            for (const channelMember of this.callView.channel.channel.callParticipants) {
+                if (this.callView.filterVideoGrid && !channelMember.isStreaming) {
+                    continue;
+                }
+                tiles.push({ channelMember: replace(channelMember) });
+            }
+            return insertAndReplace(tiles);
```

Comment:
Note: this compute is actually hiding many different concepts:
- list of participants to display in general
- list of participants that are streaming
- filter only those that are streaming

Maybe some day we would want some intermediary computes